src/util/verify-chromium-port.js

```diff
@@ -0,0 +1,52 @@
+import { createServer } from 'node:http';
+
+import { UnusablePortError } from '../errors.js';
+
+/**
+ * Determine if a port is available
+ * @param {number} port Port to test
+ * */
+async function isPortAvailable(port) {
+  return new Promise((resolve) => {
+    const server = createServer();
+    server.once('listening', () => {
+      server.close(() => resolve(true));
+    });
+    server.once('error', () => resolve(false));
+    server.listen(port);
+  });
+}
+
+/**
+ * Validate that requested port is a valid port
+ * @param {number} port Debugging port
+ * @returns {boolean}
+ */
+function isPortValid(port) {
+  if (!port) {
+    return false;
+  }
+  if (Number.isNaN(port) || !Number.isInteger(port)) {
+    throw new UnusablePortError(`Port provided is not an integer (${port})`);
+  }
+  if (port < 0 || port > 65535) {
```

Comment:
```suggestion
  if (port <= 0 || port > 65535) {
```
`0` should be excluded here. It is not reachable and if it were, it would result in a random port being chosen.